contracts/0.0.1/chatroom-slim.js

```diff
@@ -0,0 +1,666 @@
+"use strict";
```

Comment:
It's autogenerated by `esbuild`. Don't know about how to get it to remove those redefinitions.

Yes it must be part of the project, as otherwise new installations won't be able to process old messages (see PR description for more info)